src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let personState = <p>I am not married</p>;
+
+  if (person.isMarried) {
+    if (person.sex === 'm') {
+      personState = <p>{`${person.partnerName} is my wife`}</p>;
+    }
+
+    if (person.sex === 'f') {
+      personState = <p>{`${person.partnerName} is my husband`}</p>;
+    }
+  }
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```